README.md

```diff
@@ -1,2 +1,66 @@
 # FitTracker
-Service runs on port 8080
+App for tracking your weight and calorie intake.
+
+## Local Setup
+
+### Prerequisites
+
+#### Java 21
+
+  1. Navigate to https://jdk.java.net/21/ and download your platform's zip file.
+  2. Extract the Java 21 zip file to a sensible **_your_path_** on your filesystem.
+  3. Set value of **JAVA_HOME** environment variable to **<_your_path_>\jdk-21**
+  4. Add **<_your_path_>\jdk-21\bin** to **PATH** environment variable of OS.
+  5. Issue command to ensure the Java 21 install succeeded:
+    ```
+    java -version
+    ```
+
+#### Maven
+  1. Navigate to https://maven.apache.org/download.cgi and download zip file.
+  2. Extract distribution archive in a sensible **_your_path_** on your filesystem.
+3. Add \bin directory of the created directory **_your_path_\apache-maven-x.y.z** to the PATH environment variable
+4. Issue command to ensure the Maven install succeeded:
+    ```
+    mvn -v
+    ```
+#### Docker
+1. Navigate to https://www.docker.com/ and download your platform's installer.
+2. Follow installation instructions and finish the installation.
+
+#### Database
```

Comment:
So we've got `## Local Setup` and `### Prerequisites`, so this should be `### Database` (this is second part of local setups, not part of prerequisites).